content/posts/hello-world/index.md

```diff
@@ -0,0 +1,78 @@
+
+We created 2i2c so that these organizations can use entirely open-source tools without hiring and training their own dev-ops and infrastructure talent, and so that _development_ and _support_ of open tools for interactive computing continues to represent the interests of research and education.
+
+## Why a non-profit?
+
+It may sound strange to create a non-profit organization when there are many VC-funded startups and large tech companies offering notebook services these days. However, we think that a non-profit organization is the right approach to balance the interests of all the stakeholders that we wish to serve. We hope that 2i2c will be a partner to:
+
+- **Research and educational communities**, who can rely on 2i2c to provide them cutting-edge infrastructure for interactive computing that is 💯 open source.
+- **Open source communities**, who can count on 2i2c to spend its time supporting and growing the communities that underlie the tools that we deploy.
+- **Cloud providers**, who wish to help the research and educational community via their infrastructure.
+- **Supporters of open source** who wish to support interactive computing for research and education via a non-profit dedicated to exactly this mission.
+
+As a non-profit, 2i2c is dedicated to supporting an ecosystem of tools and stakeholders across the open source community, and to ensuring that those tools are well-suited for research and education. We believe strongly in mission-driven work, and non-profit status will ensure that the work that we do is always aligned with our mission and values.
+
+## What are we going to do?
+
+With all of that in mind, what is 2i2c actually going to do? We are still working out the details here, but here's a rough picture:
+
+Offer **hosted interactive computing environments on cloud infrastructure**. These will be entirely open source, and customizable for the communities that are using them. We wish to build upon the success of JupyterHub as a gateway to computational resources and environments, learning environments, and communities of users. We will offers these environments as a service for those in research and education, running on entirely open-source, vendor-agnostic, and community-driven infrastructure.
+
+Provide **collaboration and development for interactive computing in research and education**. Beyond providing hub infrastructure, there are also many ways in which solving problems in research and education can lead to better tools, infrastructure, and workflows in the open source community. For example - how can we generalize one community's needs in scalable computing so that other researchers can use the same tools? We hope that 2i2c can be an aggregator and integrator of many perspectives in research and education, and build tools that are maximally useful for these communities.
```

Comment:
I think the grammar for the question here is off just a touch I think it is more that we want to "generalize solutions" rather than "generalize needs" (needs tend to be specific, but we can abstract / generalize solutions). Maybe "How can we generalize solutions that serve one communities needs in scaleable computing so that other researchers can use the same tools?" (or similar)